draft-oid4vc-haip-sd-jwt-vc.md

```diff
@@ -356,17 +359,29 @@ The following is a non-normative example of a presentation definition for a SD-J
 
 # Crypto Suites
 
-Issuers, holders and verifiers MUST support P-256 (secp256r1) as a key type with ES256 JWT algorithm for signing and signature validation whenever this profiles requires to do so:
+Cryptography is required by the following operations:
+
+- Attestation of trust with the entities of the ecosystem.
+- Issuance and presentation using the cryptografic proof of possession of the Wallet Instance Attestations.
+- Issuance of the Verifiable Credentials.
+- Presentation Request: the process where a Verifier requests to the Holder the presentation of the Verifiable Credentials.
+- Presentation using the cryptografic proof of possession of the Credentials.
+
+With the aim to ensure a secure interoperability model for all ecosystem participants,
+Issuers, Holders, and Verifiers MUST support P-256 (secp256r1) as a key type with the ES256 JWT algorithm [@!RFC7518] for the creation and the verification of the signatures.
+This requirement is also in accordance with the following specifications, which complement the High Assurance Interoperability Profile:
+
+- SD-JWT-VC [@!I-D.ietf-oauth-sd-jwt-vc]
+- DPoP [@!RFC9449]
+- OAuth 2.0 Attestation-Based Client Authentication [@!I-D.looker-oauth-attestation-based-client-auth]
```

Comment:
I don't understand why we need this.. meaning, these drafts do not specify mandatory to implement crypto algs so really not sure what "being in accordance with these drafts" even mean
```suggestion
```